### ISSUE TITLE
Labeler incorrectly identifies established accounts that delete old posts as “New Account”

### ISSUE THREAD

**Issue by tabula-raza** [Bossett/bsky-profile-labeller#1]
Issue: The labeler is identifying accounts that have been actively posting for more than 30 days as “new account” if that account has deleted its posts older than 30 days old. 

Expected behavior: labeler labels accounts that have been established (or made its first overall post) in the last 30 days (as opposed to accounts whose oldest extant post is <=30 days old?)

Notes: this appears to become an issue where accounts that delete (or auto delete) old posts and leave a trailing post history of only the 30 most recent days, resulting in an erroneous (and persistent) “new account” label on established accounts. I expect accounts that delete old posts but leave a longer trailing history (eg 45 or 60 days) would not have this issue. I have not attempted to test the effect of changing labeler settings as none seemed directly related.

**Comment by Bossett** [comment]
Yeah - right now this is 'intended but not desired' - I'm working out how I can best identify that I've labelled something before, and not re-apply 'new account'.

I have a couple of principles I'm trying to keep to:
* an account's life begins when they first post, because sitting on an account to age it and then posting is a common spam tactic
* the labeller doesn't save state, and only labels based on things that anyone else can observe at the time of labelling

But I think I can read negated labels and then skip re-applying if I see that there's a negated (i.e. removed) label on an account, and if I can get that working I think we get everything we want.
